Day-6/src/components/GithubCard.js

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+
+const GithubCard = ({ data }) => {
+  const [teamData, setTeamData] = useState([]);
+  console.log(data);
+
+  useEffect(() => {
+    displayUserData();
+  }, []);
+
+  const fetchUserData = async (url) => {
+    const response = await fetch(url);
+    const data = response.json();
+    return data;
+  };
+
+  const displayUserData = () => {
+    const output = [];
+    data &&
+      data.map(async (user) => {
+        const url = `https://api.github.com/users/${user.username}`;
+        const userData = await fetchUserData(url);
+        output.push(userData);
+        setTeamData([...teamData, ...output]);
+      });
+  };
+
+  return (
+    <div className="card-container">
+      {teamData.map((item) => {
+        console.log(item.login);
+        return (
+          <div className="card-tag" key={item.id}>
+            <img src={item.avatar_url} alt={`${item.name || item.login} img`} />
```

Comment:
No need to write "img" here. Screen readers read it as "alt text"'s image

Reply:
okay got it ! done